### DISCUSSION TITLE
Global state in preact

### DISCUSSION THREAD

**Discussion by siduck** [preactjs/preact#3385] [Q&A]
hi guys! what do you use for global state in preact? I use this https://github.com/RisingStack/react-easy-state since it lets me export state and that can be used /edited in other files

**Comment by zgoda** [comment]
[Nanostores](https://github.com/nanostores/nanostores) but I consider this decision highly personal.

**Reply by siduck** [reply]
ive been using this so far https://github.com/RisingStack/react-easy-state
